backend/api/quivr_api/modules/sync/utils/syncutils.py

```diff
@@ -168,6 +168,8 @@ async def process_sync_file(
         ]:
             raise ValueError(f"Incompatible file extension for {downloaded_file}")
 
+        storage_path = f"{storage_path.split('.')[0]}.{storage_path.split('.')[-1]}"
```

Comment:
we would stop at everything before the "." For example
"this.is.a.test.pdf" will be -> "this.pdf" ?